### DISCUSSION TITLE
Scope of argo rollouts?

### DISCUSSION THREAD

**Discussion by manjeet-fetchai** [argoproj/argo-rollouts#1290] [Q&A]
I wanted to confirm does the argo rollouts: canary strategy is supported for backend purposes.
Will the pods be able to communicate internally with each other in same namespace with the canary rollout using any service or does it only routes traffics coming from outside to stable/canary pod.

Pod A -> Pod B/stable
Pod A -> Pod B/canary 
without specifying the defined serivce of stable/canary

**Comment by kostis-codefresh** [comment]
Argo rollouts uses the standard Kubernetes service resource behind the scenes. So if you have setup a service that is used by other internal pods, then those will "see" the canary as well. If your service is only used by external traffic, then only external traffic will "see" the canary.
